build.gradle

```diff
@@ -264,7 +265,7 @@ run {
 shadowJar {
     baseName = 'java-tron'
     classifier = null
-    version = null
+    version = '1.0.0'

```

Comment:
Not need to set a special version number. removing shadowJar.enabled could also generate java-tron.jar lib.